src/support-prompt-group/__tests__/support-prompt-group.test.tsx

```
@@ -0,0 +1,164 @@
+
+    expect(wrapper.findItems().length).toBe(3);
+  });
+
+  test("Finds item by id", () => {
+    const wrapper = renderSupportPromptGroup({});
+
+    expect(wrapper.findItemById("item-1")?.getElement()).toHaveTextContent("Item 1");
+  });
+
+  test("fires onClick", () => {
+    const wrapper = renderSupportPromptGroup({});
+
+    wrapper.findItemById("item-1")!.click();
+
+    expect(onItemClick).toHaveBeenCalledTimes(1);
+  });
+
+  test("Assigns vertical class with vertical alignment", () => {
+    const wrapper = renderSupportPromptGroup({
+      alignment: "vertical",
+    });
+
+    expect(wrapper.getElement()).toHaveClass(styles.vertical);
+  });
+
+  describe("a11y", () => {
+    test("Group has accessible name", () => {
+      const wrapper = renderSupportPromptGroup({});
+
+      expect(wrapper.getElement()).toHaveAccessibleName("Test support prompt group");
+    });
+
+    test("Prompt has accessible name", () => {
+      const wrapper = renderSupportPromptGroup({});
+
+      expect(wrapper.findItemById("item-1")!.getElement()).toHaveAccessibleName("Item 1");
+    });
+  });
+
+  describe("Keyboard navigation", () => {
+    afterEach(() => {
+      cleanup();
+    });
+    const ref: { current: SupportPromptGroupProps.Ref | null } = { current: null };
+
+    test("Arrow keys move focus in vertical alignment", () => {
+      const wrapper = renderSupportPromptGroup({}, ref);
+
+      ref.current?.focus("item-1");
+
+      fireEvent.keyDown(wrapper.getElement(), { keyCode: KeyCode.right });
+      expect(wrapper.findItemById("item-2")!.getElement()).toHaveFocus();
+
+      fireEvent.keyDown(wrapper.getElement(), { keyCode: KeyCode.down });
+      expect(wrapper.findItemById("item-3")!.getElement()).toHaveFocus();
+    });
+
+    test("Arrow keys move focus in horizontal alignment", () => {
+      const wrapper = renderSupportPromptGroup({ alignment: "horizontal" }, ref);
+
+      ref.current?.focus("item-1");
```

Here we should use `!` instead of `?`, so that we get an error message if this value is null unexpectedly, instead of the test silently continuing. Same for the other tests in this file